backend/schema/message.py

```diff
@@ -1,8 +1,10 @@
+import asyncio
+import concurrent.futures
 from datetime import datetime
 from typing import List
-
-from pydantic import AnyHttpUrl, BaseModel
-
+from fastapi import HTTPException, status
+from pydantic import AnyHttpUrl, BaseModel, Field, root_validator
```

Comment:
`Field` imported but unused. You can remove it.
git commit would not even pass with the pre-commit config. @AI-fae 😒

Reply:
fixed boss